scripts/compare_healthdata_workaround.R

```diff
@@ -0,0 +1,53 @@
+# This file is meant to AB test the workaround we used in run.R to get the most
+# recent data from healthdata.gov. This requires manually downloading the right
+# file and transforming it in the way the HHS pipeline would. All that is done
+# in `get_health_data`.
+
+source(here::here("R", "load_all.R"))
+
+forecast_generation_date <- as.Date("2023-03-01")
+
+workaround_data <- get_health_data(forecast_generation_date) %>%
+  process_healthdata()
+
+epidatr_data <- bind_rows(
+  epidatr::pub_covidcast(
+    "hhs",
+    "confirmed_admissions_influenza_1d_prop_7dav",
+    "state",
+    "day",
+    "*",
+    epirange(forecast_generation_date - 2000L, forecast_generation_date),
+    as_of = forecast_generation_date
+  ),
+  epidatr::pub_covidcast(
+    "hhs",
+    "confirmed_admissions_influenza_1d_prop_7dav",
+    "nation",
+    "day",
+    "*",
+    epirange(forecast_generation_date - 2000L, forecast_generation_date),
+    as_of = forecast_generation_date
+  )
+)
+
+# Are there differences in the overlapping (time_value, geo_keys)? No.
+full_join(
+  workaround_data,
+  epidatr_data %>% select(geo_value, time_value, value),
+  by = c("geo_value", "time_value")
+) %>%
+  filter(is.na(value.x) | is.na(value.y)) %>%
```

Comment:
?? this seems suspicious combined with the numeric comparison below.  filter isn't "filter out"

Reply:
Good catch. FWIW, these are the modified versions of comparisons Nat used when making the original workaround to make sure the data pulled from the API matches the data we get and transform from healthdata.gov. He kept them in a comment and I figured they're worth pulling into a separate script and saving for posterity. Guess I made a couple mistakes when adjusting them.

These tests tend to surface a few differences, but I think that just reflects very minor differences in our API and the healthdata site (grabbing a random date, I see 104 row differences out of 75k rows).